Assets/VRTK/Scripts/VRTK_ThirdPersonCamera.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Third Person Camera|Scripts|0090
```

Comment:
This should be in a specific directory. Nothing should be in the top level `Scripts` directory now.